components/index/standard.js

```diff
@@ -264,7 +269,7 @@ async connectTo${pascalCase(serverName)}(){ await this.connect({ servers: ["${se
       });
     }
     
-    ${serverWrapperFunctions.join('\n')}`;
+    ${renderConnectServerFunctions(Object.entries(asyncapi.servers())).join('\n')}`;

```

Comment:
For me better is passing the object (dictionary with servers and their names), rather than array of arrays.

Reply:
```suggestion
    ${renderConnectServerFunctions(asyncapi.servers()).join('\n')}`;
```

and then in the function:

```ts
for (const [serverName, server] of Object.entries(servers || {})) {
```

Reply:
You are right, that's better 👍